pkg/volume/util/subpath/subpath_linux.go

```diff
@@ -231,7 +231,7 @@ func doCleanSubPaths(mounter mount.Interface, podDir string, volumeName string)
 
 		// scan /var/lib/kubelet/pods/<uid>/volume-subpaths/<volume>/<container name>/*
 		fullContainerDirPath := filepath.Join(subPathDir, containerDir.Name())
-		err = filepath.Walk(fullContainerDirPath, func(path string, info os.FileInfo, err error) error {
+		err := filepath.Walk(fullContainerDirPath, func(path string, info os.FileInfo, initErr error) error {
 			if path == fullContainerDirPath {
 				// Skip top level directory
 				return nil
```

Comment:
(line 241) returning the error will abort filewalk and neglect sibling directories. Is that what we want?